frontend/src/components/Permissions.tsx

```diff
@@ -107,213 +157,240 @@ const Permissions: React.FC<PermissionsProps> = ({
     handlePermissionsChange({ expiresAt: expiryDate });
   };
 
-  return (
-    <div>
-      <div className="mb-6">
-        <ul className="flex flex-col w-full">
-          {capabilities?.scopes.map((scope, index) => {
-            const ScopeIcon = iconMap[scope];
-            return (
-              <li
-                key={index}
-                className={cn(
-                  "w-full",
-                  scope == "pay_invoice" ? "order-last" : "",
-                  !canEditPermissions && !permissions.scopes.has(scope)
-                    ? "hidden"
-                    : ""
-                )}
+  return !canEditPermissions ? (
```

Comment:
`canEditPermissions` is completely wrong. As we discussed, if the app sets  something, then that thing cannot be edited. No matter if it's the name, request methods, max amount, expiry, ...